docs/Documentation/The-User-Behavior.md

```diff
@@ -35,7 +35,7 @@ Configure::write('UserTools.Behavior', [/* Config goes here */]);
 * **defaultRole**: Used for a role, default is null, enter a string if you want a default role
 * **hashPassword**: Hash the password or not, default is true
 * **userActive**: Use the `active` field or not. By default users are set to active.
-* **generatePassword**: Generate and send the password instead of using token verification, default is false
+* **generatePassword**: Generate and send the password instead of using token verification, default is false. Applies to password reset only.
```

Comment:
This is not correct, it doesn't apply to the reset, in fact it's used to send the user an auto-generated password when you decide that you don't want to allow users to sign up with their own password.

```php
		if ($options['generatePassword'] === true) {
			$password = $this->generatePassword();
			$entity->{$this->_field('password')} = $password;
			$entity->clear_password = $password;
		}
```